app/templates/dashboard copy.html

```diff
@@ -0,0 +1,118 @@
+{%- from "govuk_frontend_jinja/components/tag/macro.html" import govukTag -%}
+{%- from "govuk_frontend_jinja/components/table/macro.html" import govukTable -%}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{% from "partials/start-application-button.html" import startApplicationButton %}
+{% from "partials/available_in_language.html" import available_in_language %}
+{% from "partials/round_closed_warning.html" import round_closed_warning %}
+
+{% extends "base.html" %}
+{% set pageHeading %}
+{% trans %}Your applications{% endtrans %}
+{% endset %}
+{% block content %}
+    <h1 class="govuk-heading-xl">{{ pageHeading }}</h1>
+{% if is_past_submission_deadline %}
+    {{ round_closed_warning(fund_name, round_title, submission_deadline) }}
+{% endif %}
+    <p class="govuk-body">
+        {% trans %}You have started{% endtrans %}&nbsp;{% trans count=applications|length %}{{ count }} application{% pluralize %}{{ count }} applications{% endtrans %}&nbsp;{% trans %}using this email address{% endtrans %}.
+    </p>
+{{available_in_language(inset=False, warn=True)}}
+    {% set ns = namespace(rows = []) %}
+    {% for application in applications %}
+        {% set status_class %}
+            {% if is_past_submission_deadline %}
+                {% if application.status == 'SUBMITTED' %}
+                    complete-tag
+                {% elif application.status == 'NOT_SUBMITTED' %}
+                    complete-tag
+                {% endif %}
+            {% endif %}
+            {% if not is_past_submission_deadline %}
+                {%  if application.status == 'SUBMITTED' %}
+                    complete-tag
+                {%  elif application.status == 'NOT_STARTED' %}
+                    not-started-tag
+                {%  elif application.status == 'IN_PROGRESS' %}
+                    in-progress-tag-new
+                {%  elif application.status == 'READY_TO_SUBMIT' %}
+                    in-progress-tag-new
+                {% endif %}
+            {% endif %}
+        {% endset %}
+
+        {% if not application.status == 'SUBMITTED' and not is_past_submission_deadline %}
+            {% set application_link = '<a class="govuk-link" href="{}">{}</a>'.format(url_for('application_routes.tasklist', application_id=application.id), gettext('Continue application')) %}
+        {% endif %}
+
+        {% set row = ns.rows.append(
+        [
+            {
+                'text': application.project_name if application.project_name else gettext('Untitled project')
+            },
+            {
+                'text':  govukTag({
+                    'text': application.status|snake_case_to_human ,
+                    'classes': status_class
+                })
+            },
+            {
+                'html': application.last_edited|datetime_format_short_month if application.last_edited
```

Comment:
Not sure if the if is needed due to the changes you made to the filter:

https://github.com/communitiesuk/funding-service-design-frontend/blob/fead7cfe5a9f19559a8ba06017e387846450498b/app/filters.py#L11-L19

Reply:
Just realized this in the copied file...